app/Http/Controllers/Admin/AdminController.php

```diff
@@ -20,6 +23,19 @@ public function __construct()
 
     public function index()
     {
-        return view('admin.pages.dashboard');
+        $posts = Post::selectRaw('DATE(created_at) as date, count(id) as count')
+            ->whereBetween('created_at', [now()->subDays(7), now()])
+            ->orderBy('created_at', 'ASC')
+            ->groupBy('date')
+            ->get();
+        $datas = $days = array(0, 0, 0, 0, 0, 0, 0);
```

Comment:
```suggestion
        $datas = $days = [0, 0, 0, 0, 0, 0, 0];
```
1. Mảng dùng [] ko dùng array() và cái này push vào ko đc à?

Reply:
@manhtd-1083 dạ đc ạ